ui/src/api.js

```diff
@@ -1387,7 +1389,54 @@ const Api = (req) => {
                         if (err) {
                             reject(err);
                         } else {
-                            resolve(data);
+                            let result = {
+                                workLoadData: [],
+                            };
+                            let workloadMeta = {
+                                totalDynamic: 0,
+                                totalStatic: 0,
+                                totalRecords: 0,
+                                totalHealthyDynamic: 0,
+                            };
+                            let totalHealthyDynamicCount = 0;
+                            if (data && data.workloadList != null) {
+                                workloadMeta.totalRecords =
+                                    data.workloadList.length;
+                                if (category === 'static') {
+                                    data.workloadList.forEach((workload) => {
+                                        if (workload.provider === 'Static') {
```

Comment:
can we use constant here from constants.js?

Reply:
done